spring-boot-security-taglibs/src/main/java/org/baeldung/security/HomeController.java

```diff
@@ -0,0 +1,16 @@
+package org.baeldung.security;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Controller
+@RequestMapping("/")
+public class HomeController {
+    @RequestMapping("")
+    public String home(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
`request` and `response` here are unnecessary. Let's remove them so that readers can spend more time looking at the relevant code in your article and less at the boilerplate.